src/test/ui/parser/mismatched-braces/missing-close-brace-in-impl-trait.stderr

```diff
@@ -0,0 +1,46 @@
+error: this file contains an un-closed delimiter
+  --> $DIR/missing-close-brace-in-impl-trait.rs:13:53
+   |
+LL | impl T for () {
+   |               - un-closed delimiter
+...
+LL |
+   |                                                     ^
+
+error: expected one of `async`, `const`, `crate`, `default`, `extern`, `fn`, `pub`, `type`, `unsafe`, or `}`, found `trait`
+  --> $DIR/missing-close-brace-in-impl-trait.rs:6:1
+   |
+LL | impl T for () {
+   |               - unclosed delimiter
+...
+LL | fn foo(&self) {}
+   |                 -
+   |                 |
+   |                 expected one of 10 possible tokens here
+   |                 help: `}` may belong here
```

Comment:
A hard ask: can the help note be moved to the next line? (Asking from the POV of a user. Please say firmly no if it entails a lot of compiler complexity. ^^)

Reply:
It is _possible_, but I think 

1) it'll require some extensive carrying around of the initial indentation to be anywhere close to correct and has to account or cases like `trait T { fn foo() {}` where everything is in one line, and
2) Making the suggestion go in the right place would force it to no longer be a label as it is now, but rather a new subdiagnostic window making it much more verbose.

Reply:
I'm going to take this as "It's a bad idea". :)